src/permissionGroups/components/PermissonGroupWithChannelsDetailsPage/PermissonGroupWithChannelsDetailsPage.tsx

```diff
@@ -0,0 +1,215 @@
+          toggleValue,
+          setChannelDisplayValues,
+          channelsDisplayValues,
+          channelChoices,
+        );
+
+        const handleHasRestrictedChannelsChange = () => {
+          change({
+            target: {
+              name: "hasRestrictedChannels",
+              value: !data.hasRestrictedChannels,
+            },
+          });
+        };
+
+        return (
+          <DetailPageLayout>
+            <TopNav
+              href={permissionGroupListUrl()}
+              title={permissionGroup?.name}
+            />
+            <DetailPageLayout.Content>
+              <PermissionGroupInfo
+                data={data}
+                disabled={disabled}
+                errors={errors}
+                onChange={change}
+              />
+              <FormSpacer />
+              <PermissionGroupMemberList
+                disabled={disabled}
+                {...listProps}
+                users={data?.users || []}
+              />
+            </DetailPageLayout.Content>
+            <DetailPageLayout.RightSidebar>
+              <Box display="flex" flexDirection="column" height="100%">
+                <Box overflow="hidden" __maxHeight="50%">
+                  <AccountPermissions
+                    permissionsExceeded={permissionsExceeded}
+                    data={data}
+                    disabled={disabled}
+                    permissions={permissions}
+                    onChange={change}
+                    errorMessage={permissionsError}
+                    fullAccessLabel={intl.formatMessage({
+                      id: "mAabef",
+                      defaultMessage: "Group has full access to the store",
+                      description: "checkbox label",
+                    })}
+                    description={intl.formatMessage({
+                      id: "CYZse9",
+                      defaultMessage:
+                        "Expand or restrict group's permissions to access certain part of saleor system.",
+                      description: "card description",
+                    })}
+                  />
+                </Box>
+                <Box overflow="hidden" __maxHeight="50%">
+                  <ChannelPermission
+                    channelsDisplayValues={channelsDisplayValues}
+                    allChannels={channels}
+                    hasRestrictedChannels={data.hasRestrictedChannels}
+                    selectedChannels={data.channels}
+                    onHasRestrictedChannelsChange={
+                      handleHasRestrictedChannelsChange
+                    }
+                    onChannelChange={handleChannelChange}
+                    disabled={disabled}
+                  />
+                </Box>
+              </Box>
+            </DetailPageLayout.RightSidebar>
+            <div>
+              <Savebar
+                onCancel={() => navigate(permissionGroupListUrl())}
```

Comment:
**question:** on many views we drop this button, we have backbutton anyway, is this necessary?

Reply:
This is not in task scope, and this component is copy pase from old page because I use feature flags